docs/pages/database-access/guides/postgres-cloudsql.mdx

```diff
@@ -0,0 +1,336 @@
+This will let Teleport Database Service generate an IAM auth token when
+connecting to the database as a service account user we created above.
+
+If creating a new one, go to the [Service Accounts](https://console.cloud.google.com/iam-admin/serviceaccounts)
+page and create another service account:
+
+![Create Service Account](../../../img/database-access/guides/cloudsql/service-account-db-service.png)
+
+Assign it the "Service Account Token Creator" role:
+
+![Grant Service Account Token Creator to Service Account](../../../img/database-access/guides/cloudsql/service-account-db-service-grant.png)
+
+Once created, go to that service account's Keys tab and create a new key:
+
+![Service Account Keys](../../../img/database-access/guides/cloudsql/service-account-keys.png)
+
+Make sure to choose JSON format:
+
+![Service Account New Key](../../../img/database-access/guides/cloudsql/service-account-new-key.png)
+
+Save the file, your Teleport Database Service will need it to be able to generate
+IAM auth tokens.
+
+## Gather Cloud SQL Instance Information
+
+To connect a Cloud SQL database to Teleport, you'll need to gather a few pieces
+of information about the instance.
+
+- GCP Project ID.
+
+You can normally see it in the organization view at the top of the GCP dashboard.
+
+- Cloud SQL instance ID.
+
+The instance ID is the name of your Cloud SQL instance shown at the top of the
+Overview page:
+
+![Instance ID](../../../img/database-access/guides/cloudsql/instance-id.png)
+
+- Cloud SQL instance endpoint.
+
+You will use the instance's public IP address to connect to it. It can be viewed
+on the "Connect to this instance" panel on the Overview page:
+
+![Instance Public IP](../../../img/database-access/guides/cloudsql/instance-public-ip.png)
+
+- Cloud SQL instance root certificate.
+
+The instance's root certificate is required so Teleport can validate the
+certificate presented by the database instance. You can download `server-ca.pem`
+file from the Connections tab under Security section:
+
+![Instance Root Certificate](../../../img/database-access/guides/cloudsql/instance-root-ca.png)
+
+## Configure Teleport
+
+Cloud SQL support for Teleport Database Access is available starting from
+`6.1` release.
+
+Download the appropriate version of Teleport for your platform from
+our [downloads page](https://goteleport.com/teleport/download).
+
+Follow the installation [instructions](../../installation.mdx).
+
+### Start Auth/Proxy Service
+
+Create a configuration file for a Teleport service that will be running
+auth and proxy servers:
+
+```yaml
+teleport:
+  data_dir: /var/lib/teleport
+  nodename: test
+auth_service:
+  enabled: "yes"
+proxy_service:
+  enabled: "yes"
+  # Set public address proxy will be reachable at.
+  public_addr: teleport.example.com:3080
+  # The PostgreSQL proxy listens on the regular web proxy port.
+  web_listen_addr: 0.0.0.0:3080
+ssh_service:
+  enabled: "no"
+```
```

Comment:
I would suggest we use 'Getting started with Teleport' as the canonical reference for setting up Teleport.

Reply:
These are supposed to be self-contained guides, incl. setting up Teleport. This would also make it inconsistent with the other guides.

Reply:
In that case, I would suggest we need to setup TLS for the proxy.

Reply:
Yeah, we can use teleport configure and ACME here like in quickstart. I'll update.

Reply:
Updated.